tracpro/polls/tasks.py

```diff
@@ -48,8 +47,8 @@ def org_task(self, org):
 
         total_runs = 0
         for poll in Poll.objects.active().by_org(org):
-            poll_runs = client.get_runs(flows=[poll.flow_uuid], after=last_time, before=until)
-            total_runs += len(poll_runs)
+            poll_runs = client.get_runs(flow=poll.flow_uuid, after=last_time, before=until)
+            total_runs += len(poll_runs.all())
```

Comment:
This fails without the .all(). I was getting a failure from the celery task yesterday when I pushed this out to staging. The .all() isn't needed elsewhere but it is needed to calculate the len().